contrib/platform/src/com/sun/jna/platform/win32/Advapi32.java

```diff
@@ -1558,4 +1561,153 @@ public boolean AccessCheck(Pointer pSecurityDescriptor,
                                PRIVILEGE_SET PrivilegeSet,
                                DWORDByReference PrivilegeSetLength,
                                DWORDByReference GrantedAccess, BOOLByReference AccessStatus);
+	
+	/**
+	 * Encrypts a file or directory. All data streams in a file are encrypted. All
+	 * new files created in an encrypted directory are encrypted.
+	 *
+	 * @param lpFileName
+	 *         The name of the file or directory to be encrypted.
+	 * @return If the function succeeds, the return value is nonzero. If the
+	 * function fails, the return value is zero. To get extended error
+	 * information, call GetLastError.
+	 */
+	public boolean EncryptFileW(WString lpFileName);
```

Comment:
to be consistent with the rest of JNA, these should be declared without the `W`, since we will never support the non-W version, see `GetFileSecurity` above for example.